_posts/2020-06-20-Basic-Querying.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: post
+date: 2020-06-20 18:00:00 +010
+title: Basic Querying and more on the Cíannachta Glinne Geimen
+author: et
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+## Introduction
+In our [previous blogpost](https://cyocum.github.io/2020/06/06/Manuscript-Variants-and-the-Origins-of-C%C3%ADannachta-Glinne-Geimen.html), we looked at the variants between and within manuscript versions of the medieval Irish genealogies that we can encounter while working on IrishGen and how we go about encoding the information involved. This was in the context of the genealogies' account of the origins of the kingdom of the Cíannachta Glinne Geimen. In this post, we're going to continue following the history of this kingdom but, this time, consider some basic examples of how the IrishGen database can be queried.
+
+For introductory reading on the genealogies, we recommend Nollaig Ó Muraíle’s published lecture, _[The Irish Genealogies: Irish History’s Poor Relation?](https://www.ria.ie/its-occasional-lecture-vol-3-irish-genealogies-irish-historys-poor-relation)_, and Matthew Holmberg’s PhD thesis, ‘[Towards a Relative Chronology of the Milesian Genealogical Scheme](https://dash.harvard.edu/handle/1/37945002)’ (pp. 3–54). For an overview of IrishGen and our use of RDF, see the project [README](https://github.com/cyocum/irish-gen). All our data is available in our [GitHub repository](https://github.com/cyocum/irish-gen).
+
+## Sources
+The [Book of Leinster](https://www.vanhamel.nl/codecs/Dublin,_Trinity_College,_MS_1339) (**L**) and [Rawl. B.502](https://www.vanhamel.nl/codecs/Oxford,_Bodleian_Library,_MS_Rawlinson_B_502) (**R**) continue to be our main manuscript sources, as these contain the only genealogical collections currently in the IrishGen database that include significant quantities of material on the Cíannachta. The third major genealogical collection in IrishGen, that from [Laud Misc. 610](https://www.vanhamel.nl/codecs/Oxford,_Bodleian_Library,_MS_Laud_Misc._610) (**Ld**), also makes an appearance.
+
+A table of texts from the genealogies cited in this post is given at the end.
+
+## The Sons of Findchán
+We have seen, notwithstanding isolated variants, that the genealogies (Text 1) hold that the first descendant of Tadg mac Céin to acquire territory in Glengiven was Findchán, son of Fíac, son of Findchad Ulach and that he had eight sons. We can check the statement that he had eight sons against the rest of the IrishGen database. This can be done using a visual explorer, such as that provided by *[GraphDB](https://www.ontotext.com/try-graphdb-se/)*, which was used to produce the visualizations in the last blogpost. It can also be done using a [SPARQL](https://www.w3.org/TR/rdf-sparql-query/) query:
```

Comment:
You may also want to link to the section in the GraphDB documentation that discusses the visual explorer just in case someone wants to dive into that topic.

Reply:
Is this what you had in mind? http://graphdb.ontotext.com/documentation/9.2/free/exploring-data.html